docs/docs/ProgrammingGuide/quantization-support.md

```diff
@@ -0,0 +1,68 @@
+## Introduction
+
+Quantization is a method that will use low-precision caculations to substitute float caculations. It will improve the inference performance and reduce the size of model with 3/4.
+
+## Quantize the pretrained model
+
+BigDL supports converting the pretrained model to quantized model with parameter `--quantize true`.
```

Comment:
BigDL provide command line tools for converting the pretrained (BigDL, Caffe, Torch or Tensorflow) model to ...